frontend/src/components/worksheets/WorksheetItemList.js

```diff
@@ -325,6 +326,17 @@ class WorksheetItemList extends React.Component {
                         // the file hasn't changed)
                         onUploadFinish={(e) => this.setState({ newUploadKey: Math.random() + '' })}
                     />
+                    <ImageEditor
+                        key={this.state.newUploadKey + 1}
+                        after_sort_key={getAfterSortKey(focusedItem, this.props.subFocusIndex)}
+                        worksheetUUID={info.uuid}
+                        reloadWorksheet={this.props.reloadWorksheet}
+                        onUploadFinish={(e) => this.setState({ newUploadKey: Math.random() + '' })}
+                        addImageDisplay={this.props.addImageDisplay}
```

Comment:
addimagedisplay isn't used in the ImageEditor prop. Can you check for unused props and remove them?

Reply:
thanks for pointing it out. solved.